src/components/header/HeaderActions.vue

```diff
@@ -24,7 +24,9 @@ const handleLogin = () => {
 <template>
   <div class="actions">
     <div class="actions__buttons">
-      <BaseButton type="secondary" to="#">Register</BaseButton>
+      <BaseButton type="secondary" :to="{ name: 'RegisterPage' }"
+        >Register</BaseButton
+      >
```

Comment:
Looks like there is something wrong with formatting here no?

Reply:
fixed